### PR TITLE
WIP: Documentation

### PR DESCRIPTION
**Pull request by annymsMthd**
I'm not the greatest wordsmith so I would like to ask for your help in this initial documentation for kafmesh.

### CODE REVIEW

**Comment by cuevaskoch** [comment]
This is awesome!! Going to do a copy-editing pass on it

**Comment by cuevaskoch** [comment]
Sorry, I misunderstood what "convert to draft" does

**Comment by annymsMthd** [comment]
note: Document kubernetes annotations for scraping

File: README.md [hunk 1]
```diff
@@ -1 +1,136 @@
...
+  a strongly typed wrapper around the [Goka
+  Processor](https://github.com/lovoo/goka#concepts) concept.
+
+* **View**
+
+  A view is a look into what kafka is storing for a particular topic. It takes a feed from a kafka topic and provides a key/value datastore. This is a strongly typed wrapper aroudn the [Goka View](https://github.com/lovoo/goka#concepts) concept.
+
+* **ViewSource**
+
+  View source will keep a kafka compacted topic in sink with an external source.
```
Comment (dkaminski) [inline README.md:58]:
```suggestion
  View source will keep a kafka compacted topic in sync with an external source.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,136 @@
...
+description: A example kafmesh service.
+
+output:
+  package: definitions
+  path: internal/definitions
+  module: example-service
+
+messages:
+  protobuf:
+    - ../protos
```
Comment (dkaminski) [inline README.md:90]: Is it worth showing the protobuf definition of the `userId.click` source message, or is that outside the scope of the example?

File: README.md [hunk 3]
```diff
@@ -1 +1,136 @@
...
+
+## Acknowledgments
+
+* [goka](https://github.com/lovoo/goka) A big shout out for goka doing the hard part of building out a kafka streaming service for go.
+
+* [Visual Studio Code](https://code.visualstudio.com/) for just being an all around great editor
+
+[Kafka]: https://kafka.apache.org/
+[Goka]: https://github.com/lovoo/goka
+[kafmesh-example]: https://github.com/syncromtaics/kafmesh-example
```
Comment (dkaminski) [inline README.md:136]: Typo "syncromtaics"

**Comment by thzinc** [comment]
Going to merge this and continue work on docs

**Pull request merged**